packages/app/prisma-utils/test/DbCleaner.ts

```diff
@@ -5,12 +5,12 @@ export enum DB_MODEL {
 	item2 = 'item2',
 }
 
-export async function cleanTables(prisma: PrismaClient, modelNames: readonly DB_MODEL[]) {
-	const delegates = modelNames.map<{ deleteMany: () => Promise<unknown> }>(
-		(modelName) => prisma[modelName],
-	)
-
-	for (const delegate of delegates) {
-		await delegate.deleteMany()
+export async function cleanTables(
```

Comment:
DbCleaner isn't currently exported from prisma-utils.
Considering that we no longer rely on Prisma models for truncation, we probably can export this as a generic reusable DB cleaner.
If we want to keep it type-safe, we probably can wrap it into a class, which would accept a list of supported modules as a constructor param.